spark_board/plan_extractor/transformation_node_builders/repository.py

```diff
@@ -0,0 +1,114 @@
+from typing import Callable, Dict, Optional
+
+from .aggregate_node_builder import AggregateNodeBuilder
+from .alias_node_builder import AliasNodeBuilder
+from .deduplicate_node_builder import DeduplicateNodeBuilder
+from .except_node_builder import ExceptNodeBuilder, ExceptAllNodeBuilder
+from .expand_node_builder import ExpandNodeBuilder
+from .filter_node_builder import FilterNodeBuilder
+from .generate_node_builder import GenerateNodeBuilder
+from .intersect_node_builder import IntersectNodeBuilder, IntersectAllNodeBuilder
+from .join_node_builder import JoinNodeBuilder
+from .limit_node_builder import LimitNodeBuilder
+from .logical_rdd_node_builder import LogicalRDDNodeBuilder
+from .logical_relation_node_builder import LogicalRelationNodeBuilder
+from .project_node_builder import ProjectNodeBuilder
+from .repartition_node_builder import RepartitionNodeBuilder
+from .sample_node_builder import SampleNodeBuilder
+from .sort_node_builder import SortNodeBuilder
+from .transformation_node_builder import TransformationNodeBuilder
+from .union_node_builder import UnionNodeBuilder
+from .unknown_node_builder import UnknownNodeBuilder
+from .window_node_builder import WindowNodeBuilder
+
+"""
+Returns the default set of objects that can parse each transformation sub-tree.
+"""
```

Comment:
the doc string goes inside the function (right after the `def`)

Reply:
Oops, java habit...